guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+[Silent payments](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki) is a protocol the uses static addresses to simplify payment experience while preserving privacy. Once the receiver shares a static address, senders derive a new unique on-chain address with it they make a payment. This prevents [address reuse](/guide/glossary/address/#address-reuse) without repeated user interaction.
+
+For e.g: Alice can simply post a static address on her website, and receive bitcoin donations at unique on-chain addresses. The static address itself never shows up on-chain.
+
+Silent payments also allow users to customize their static address with labels that are detected when payments are received. This is a powerful tool that provides actionable information for [coin selection](/guide/how-it-works/silent-payments/#coin-selection) with minimal manual effort.
+
+Static addresses & labels enable robust [contacts](/guide/how-it-works/silent-payments/#contacts) features. The result is a better interaction model centred around users.
+
+
+### Rationale
+
+Since the bitcoin blockchain is public, reusing an on-chain address informs the network that these payments are made to the same user. However, specifying a new address for each transaction usually requires user interaction. This is error-prone, takes time and requires manual effort.
+
+Silent payments circumvent both these issues by using static addresses (starting with `sp1`).
+
+## How silent payments work
+
+Silent payments are made in 4 broad steps:
+1. The receiver shares/publishes a static payment address (manual, one-time)
+2. The sender obtains & uses it to derive a unique on-chain address (partly automated)
+3. The sender broadcasts a transaction that pays this derived address (manual)
+4. The receiver scans the blockchain & identifies their payments (automated)
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work-mobile@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payments-work@2x.png"
+   alt-text = "An image that illustrates how silent payments work in 4 steps."
+   caption = "Alice posts a static address on the website of her non-profit, Chasing Rabbits. Bob uses it to send bitcoin from his wallet. This transaction pays a derived on-chain address that Alice can recognise while scanning the blockchain."
+   width = 1024
+   height = 640
+   layout = "full-width"
+%}
+
+## A new conceptual model
+
+Silent payments introduce a new technical model that separates payment information based on their purpose.
```

Comment:
I don't understand this sentence. How is payment information separated, what payment information is this (amount?) and from what is it separated from?

Reply:
The table and the text below it explains this...